package.json

```diff
@@ -53,18 +53,18 @@
     "pinia": "^2.0.35",
     "pinia-class": "^0.0.3",
     "regenerator-runtime": "^0.13.11",
-    "sbc-common-components": "3.0.8",
+    "sbc-common-components": "4.0.0",
     "uuid": "^9.0.0",
-    "vue": "^3.3.4",
+    "vue": "3.2.31",
     "vue-affix": "^0.5.2",
     "vue-hotjar": "^1.4.0",
     "vue-observe-visibility": "^1.0.0",
-    "vue-router": "^3.6.5",
+    "vue-router": "^4.0.3",
     "vue-the-mask": "^0.11.1",
     "vue2-filters": "^0.14.0",
     "vuelidate": "0.6.2",
-    "vuetify": "^3.3.19",
-    "vuex": "^3.6.2"
+    "vuetify": "^3.1.8",
+    "vuex": "^4.0.0"
```

Comment:
I don't think you need this anymore?

Reply:
We kept Vuex around for sbc-common-components. No longer needed?

Reply:
Yup, no longer needed. There's a comment I posed in the `index.ts` in the store folder. I probably should've put it here.

Reply:
You've removed it. I'm resolving this comment.